uint8 id
uint16 p_gain
uint16 i_gain
uint16 d_gain
uint16 profile_velocity
uint16 current_limit
uint8 send_data_flag
uint8 external_encoder_flag
uint16 joint_resolution
uint16 servo_resolution

